src/app/models/trip.js

```diff
@@ -0,0 +1,67 @@
+import Backbone from 'backbone';
+
+const Trip = Backbone.Model.extend({
+  urlRoot: 'https://ada-backtrek-api.herokuapp.com/trips',
+
+  initialize: function(attributes) {
+      // console.log(`In initialize: for the book ${ this.get('title') }`);
+    },
+
+  validate(attributes) {
```

Comment:
This is good, but you also need to test for:

1.  Continents that are valid (Asia, Africa etc).
1.  Valid numeric values for weeks and cost